### PR TITLE
Misc improvement

### PR DESCRIPTION
**Pull request by yann300**
- easy switch to goerli (by setting the chain id to 5)
 - fix printing allowed minting && add warning msg to the mint action
 - improve loadging NftCard (using Alchemy)

### CODE REVIEW

**Review by joeizang — COMMENTED** [review]
It does break the browseBage page for me. Can't even load the page normally. This is what I get.
Screenshot from 2022-07-10 00-53-54

**Comment by yann300** [comment]
oh that's right indeed, I just fixed that.

**Review by joeizang — APPROVED** [review]
I can confirm that the site now works as it should locally. Very well done

File: packages/react-app/src/components/MintingActions.jsx [hunk 1]
```diff
@@ -34,11 +35,18 @@ export default function MintingActions({ contractRef }) {
...
+    try {
+      setMessage('Please approve the transaction and wait for the validation')
+      let mintTx = await contract.publicMint(receiverAddress)
+      // console.log({ mintTx })
+      await mintTx.wait()
+      setMessage('Transaction validated')
+    } catch (e) {
+      setMessage('error while sending the transaction. ' + e.message)
+    }   
+    setTimeout(() => setMessage(''), 10000)
```
[the remark this answers is not in the document]
Comment (yann300) [reply packages/react-app/src/components/MintingActions.jsx:49]: just to clear the message panel aftere some time

**Pull request merged**